docs/entities/blocks/block.schema.json

```diff
@@ -7,6 +7,8 @@
     "canonical",
     "height",
     "hash",
+    "stacks_block_time",
+    "stacks_block_time_iso",
```

Comment:
Could a better name be just `block_time`? Given that we use `block_` prefix in all other Stacks block properties

Reply:
Yep! Done